src/models/pretraining.py

```diff
@@ -0,0 +1,99 @@
+import torch.nn as nn
+import torch
+from transformers import BertModel, BertConfig
+import numpy as np
+from torch.nn import CTCLoss
+import lpips
+
+from src.utils.common import PositionalEncoding, compute_ctc_loss
+from src.utils.masking import SpanMaskingGenerator
+from src.models.vtr.ocr import OCRHead
+
+
+class Pretrain(nn.Module):
+    def __init__(
+        self,
+        emb_size: int = 512,
+        n_head: int = 8,
+        n_layers: int = 4,
+        device: str = "cpu",
+        ocr: OCRHead = None,
+        char2int_dict: dict = None,
```

Comment:
It isn't a good practice to add the type of the variable to its name: 
https://google.github.io/styleguide/pyguide.html#3161-names-to-avoid (the last point)

You've already had this information in the typing